writeengine/wrapper/writeengine.cpp

```diff
@@ -1108,6 +1129,7 @@ int WriteEngineWrapper::insertColumnRecs(const TxnID& txnid,
             }
 
             //mark the extents to invalid
+            cout << "setting extents marks to invalid values, " << cpinfoList.size() << " cpinfos total." << endl;
```

Comment:
I believe we don't want to expose this to prod.

Reply:
Got rid of this and some other printfs/cout << lines.